components/speedreader/resources/panel/components/main-panel/index.tsx

```diff
@@ -3,40 +3,93 @@ import * as React from 'react'
 import * as S from './style'
 import Toggle from '$web-components/toggle'
 import { FontStyleList, ContentList } from '../lists'
-import ZoomControl from '../zoom-control'
+import FontSizeControl from '../font-size-control'
 import ThemeControl from '../theme-control'
+import { SiteSettings, Theme, FontSize, ContentStyle, FontFamily } from '../../api/browser'
+import BackgroundSVG from '../../svg/background'
+import { getLocale, splitStringForTag } from '$web-common/locale'
+
+interface MainPanelProps {
+  siteSettings: SiteSettings
+  onThemeChange: (theme: Theme) => void
+  onFontSizeChange: (fontSize: FontSize) => void
+  onContentStyleChange: (contentStyle: ContentStyle) => void
+  onFontFamilyChange: (fontFamily: FontFamily) => void
+  onToggleChange: (isOn: boolean) => void
+}
+
+function MainPanel (props: MainPanelProps) {
+  const braveSpeedreaderOffText = splitStringForTag('braveSpeedreaderTurnOffDesc')
+
+  let headerTextElement = null
+  let mainContentElement = (
+    <>
+        <S.Section>
+          {braveSpeedreaderOffText.beforeTag}
+          <a href="brave://settings/">{braveSpeedreaderOffText.duringTag}</a>
+          {braveSpeedreaderOffText.afterTag}
+        </S.Section>
+        <S.Bg><BackgroundSVG /></S.Bg>
+      </>
+  )
+
+  if (props.siteSettings.isEnabled) {
+    headerTextElement = (
+      <S.SiteTitleBox>
+        {getLocale('braveSpeedreaderAlwaysLoadLabel')}
+        {' '}
+        <S.SiteName>{props.siteSettings.host}</S.SiteName>
```

Comment:
This needs to use a placeholder and string replacement due to different language syntaxes

Reply:
done